backend/video/views.py

```diff
@@ -78,12 +79,46 @@ def get_video(request):
 
     # Convert audio only to boolean
     is_audio_only = is_audio_only.lower() == "true"
-
     if url is None:
         return Response(
             {"error": "Video URL not provided in query params."},
             status=status.HTTP_400_BAD_REQUEST,
         )
+    
+    ## TEMP: Handle audio_only files separately for google drive links 
+    if "drive.google.com" in url and is_audio_only:
+
+        # Construct a direct download link from the google drive url 
+        # get the id from the drive link 
+        file_id = url.split("/")[-2]
```

Comment:
This works only if the URL is of the following structure:
```
https://drive.google.com/file/d/1qJzF0CFH9atD4z32EMqsA7--aCD5kTt9/view
```

It won't work even if it is like this:
```
https://drive.google.com/file/d/1qJzF0CFH9atD4z32EMqsA7--aCD5kTt9
```

---

There can also be other formats, like:
```
https://drive.google.com/uc?id=1qJzF0CFH9atD4z32EMqsA7--aCD5kTt9
```

Can you please handle these cases? Also, if possible, explore what other formats it could be in

Reply:
Basically, a way to standardize and store the normalized URL so that we don't create different records for same video (but with different link formats)

Reply:
I have dealt with a few cases in the `utils.extract_google_drive_link_id` which can be reviewed.